2021-12-14 13:10:12,592 INFO    MainThread:8205 [wandb_setup.py:_flush():71] setting env: {}
2021-12-14 13:10:12,592 INFO    MainThread:8205 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-14 13:10:12,592 INFO    MainThread:8205 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-14 13:10:12,592 INFO    MainThread:8205 [wandb_init.py:_log_setup():371] Logging user logs to /home/matteo/NeuroGEN/TorVergataExperiments/Generative/wandb/run-20211214_131012-33dnw06r/logs/debug.log
2021-12-14 13:10:12,593 INFO    MainThread:8205 [wandb_init.py:_log_setup():372] Logging internal logs to /home/matteo/NeuroGEN/TorVergataExperiments/Generative/wandb/run-20211214_131012-33dnw06r/logs/debug-internal.log
2021-12-14 13:10:12,593 INFO    MainThread:8205 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f5d945cba60>
2021-12-14 13:10:12,593 INFO    MainThread:8205 [wandb_init.py:init():404] calling init triggers
2021-12-14 13:10:12,593 INFO    MainThread:8205 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'dataset': 'celebA', 'type': 'GAN', 'encoder_architecture': ListWrapper([(0, 32), (0, 64), (1, 128), (1, 128), (1, 256)]), 'decoder_architecture': ListWrapper([(0, 256), (0, 128), (1, 128), (1, 64), (1, 32)]), 'target_shape': (128, 128, 3), 'latent_dim': 512, 'epochs': 250, 'BS': 512, 'init_lr': 5e-05, 'nota': 'DGX'}
2021-12-14 13:10:12,593 INFO    MainThread:8205 [wandb_init.py:init():449] starting backend
2021-12-14 13:10:12,593 INFO    MainThread:8205 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-14 13:10:12,615 INFO    MainThread:8205 [backend.py:ensure_launched():199] starting backend process...
2021-12-14 13:10:12,635 INFO    MainThread:8205 [backend.py:ensure_launched():204] started backend process with pid: 8898
2021-12-14 13:10:12,636 INFO    MainThread:8205 [wandb_init.py:init():458] backend started and connected
2021-12-14 13:10:12,639 INFO    MainThread:8205 [wandb_run.py:_label_probe_notebook():917] probe notebook
2021-12-14 13:10:12,640 INFO    MainThread:8205 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-12-14 13:10:12,640 INFO    MainThread:8205 [wandb_init.py:init():519] updated telemetry
2021-12-14 13:10:12,686 INFO    MainThread:8205 [wandb_init.py:init():549] communicating current version
2021-12-14 13:10:13,352 INFO    MainThread:8205 [wandb_init.py:init():554] got version response 
2021-12-14 13:10:13,352 INFO    MainThread:8205 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-14 13:10:13,572 INFO    MainThread:8205 [wandb_init.py:init():592] starting run threads in backend
2021-12-14 13:10:14,570 INFO    MainThread:8205 [wandb_run.py:_console_start():1816] atexit reg
2021-12-14 13:10:14,570 INFO    MainThread:8205 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-14 13:10:14,571 INFO    MainThread:8205 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-14 13:10:14,571 INFO    MainThread:8205 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-14 13:10:14,572 INFO    MainThread:8205 [wandb_init.py:init():619] run started, returning control to user process
2021-12-14 13:10:14,574 INFO    MainThread:8205 [wandb_init.py:_pause_backend():284] pausing backend
2021-12-14 13:10:14,574 INFO    MainThread:8205 [jupyter.py:save_ipynb():377] looking for notebook: None
2021-12-14 13:10:14,580 INFO    MainThread:8205 [wandb_init.py:_resume_backend():293] resuming backend
2021-12-14 13:10:15,305 INFO    MainThread:8205 [wandb_init.py:_pause_backend():284] pausing backend
2021-12-14 13:10:15,305 INFO    MainThread:8205 [jupyter.py:save_ipynb():377] looking for notebook: None
2021-12-14 13:10:15,309 INFO    MainThread:8205 [wandb_init.py:_resume_backend():293] resuming backend
2021-12-14 13:10:15,453 INFO    MainThread:8205 [wandb_init.py:_pause_backend():284] pausing backend
2021-12-14 13:10:15,454 INFO    MainThread:8205 [jupyter.py:save_ipynb():377] looking for notebook: None
2021-12-14 13:10:15,457 INFO    MainThread:8205 [wandb_init.py:_resume_backend():293] resuming backend
2021-12-14 13:10:16,549 INFO    MainThread:8205 [wandb_init.py:_pause_backend():284] pausing backend
2021-12-14 13:10:16,549 INFO    MainThread:8205 [jupyter.py:save_ipynb():377] looking for notebook: None
2021-12-14 13:12:13,350 INFO    MainThread:8205 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2021-12-14 13:12:13,351 INFO    MainThread:8205 [wandb_run.py:_restore():1758] restore
2021-12-14 13:12:13,722 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 4736
  total_bytes: 4736
}

2021-12-14 13:12:13,928 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 4736
  total_bytes: 4736
}

2021-12-14 13:12:14,596 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4736
  total_bytes: 9413
}

2021-12-14 13:12:14,697 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4736
  total_bytes: 10185
}

2021-12-14 13:12:14,798 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4736
  total_bytes: 10185
}

2021-12-14 13:12:14,899 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,000 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,101 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,202 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,303 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,403 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

2021-12-14 13:12:15,735 INFO    MainThread:8205 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10185
  total_bytes: 10185
}

